.github/workflows/maintainer_merge_review.yml

```diff
@@ -1,13 +1,13 @@
-name: Maintainer merge
+name: Maintainer merge (review)
 
 on:
-  pull_request_review:
+  pull_request_review_comment:
     types: [submitted]
```

Comment:
```suggestion
    types: [created, edited]
```

Reply:
Will the bot respond if I do this?

maintainer merge